File: py/Level1/x만큼 간격이 있는 n개의 숫자.py
#
# [level 1] x만큼 간격이 있는 n개의 숫자
# https://school.programmers.co.kr/learn/courses/30/lessons/12954

# 결과
# 정확성: 100.0
# 합계: 100.0 / 100.0

def solution(x, n):
    answer = []
    y = x
    for i in range(n):
        answer.append(y)
        y += x

    return answer

File: js/lv1/x만큼 간격이 있는 n개의 숫자.js
//[level 1] x만큼 간격이 있는 n개의 숫자 12954
//https://school.programmers.co.kr/learn/courses/30/lessons/12954

//결과 
//정확성: 100.0
//합계: 100.0 / 100.0

function solution(x, n) {
    var answer = [];
    for (let i = 1; i<=n; i++)
        answer.push(x*i);
    return answer;
}